### ISSUE TITLE
Bug: Default php.ini files are missing size modifier.

### ISSUE THREAD

**Issue by patrickgarsow** [litespeedtech/cyberpanel#562]
**Observations:** php.ini values for entries like post_max_size and upload_max_filesize are set to just a number with no size modifier.

This causes problems as seen in **Issue#561** and **Issue#559**

https://github.com/usmannasir/cyberpanel/issues/561

https://github.com/usmannasir/cyberpanel/issues/559

Description of what is missing can be seen from PHP documentation: https://www.php.net/manual/en/function.ini-get.php

**Request:** Fix/Resolve Default php.ini files to have correct entries or entries that are defaulted to higher values for CyberPanel.

**Comment by usmannasir** [comment]
```[root@test1 lsws]# grep -riF "post_max_size" --include=\*.ini
lsphp71/etc/php.ini:post_max_size = 8M
lsphp72/etc/php.ini:post_max_size = 8M
lsphp70/etc/php.ini:post_max_size = 8M
lsphp55/etc/php.ini:post_max_size = 8M
admin/conf/php.ini:post_max_size = 8M
admin/misc/php.ini:post_max_size = 8M
lsphp80/etc/php.ini:post_max_size = 8M
lsphp53/etc/php.ini:post_max_size = 8M
lsphp73/etc/php.ini:post_max_size = 8M
lsphp54/etc/php.ini:post_max_size = 8M
lsphp56/etc/php.ini:post_max_size = 8M
lsphp74/etc/php.ini:post_max_size = 8M```

This is centos7. what OS are you on?

**Comment by patrickgarsow** [comment]
@usmannasir I am using CentOS Linux version 8.3.2011

**Comment by usmannasir** [comment]
let me check

**Comment by qtwrk** [comment]
```
[root@test1 lsws]# grep -riF "post_max" --include=\*.ini
lsphp71/etc/php.ini:post_max_size = 8M
lsphp74/etc/php.ini:post_max_size = 8M
lsphp73/etc/php.ini:post_max_size = 8M
admin/conf/php.ini:post_max_size = 8M
admin/misc/php.ini:post_max_size = 8M
lsphp80/etc/php.ini:post_max_size = 8M
lsphp72/etc/php.ini:post_max_size = 8M
```

just tested on centos 8 , seems alright to me 

what's the provider ? 

and please try 

`yum install http://rpms.litespeedtech.com/centos/litespeed-repo-1.2-1.el8.noarch.rpm`

to install LiteSpeed repo , then 

`yum install openlitespeed`

this should install lsphp73 ,  you can check php73's ini first , see what it shows

**Comment by patrickgarsow** [comment]
@qtwrk 
what's the provider ? - No provider...on custom lab config on vmware.

@usmannasir I created a lab server install and everything is correct so I can only believe that there was something wrong with the files when I downloaded the original server. We can close this as I have verified on CentOS 7 and 8 that the implementation works as expected now and php.ini entries have a size reference.
